src/bin/vhost_user_fs.rs

```diff
@@ -295,8 +295,8 @@ fn main() {
                 .min_values(1),
         )
         .arg(
-            Arg::with_name("sock")
-                .long("sock")
+            Arg::with_name("socket")
+                .long("socket")
```

Comment:
This is the only one where I think we should temporarily support both parameters.

Reply:
Thanks for the comments and pointers. I just updated the PR, and that seems to serve the purpose. Please let me know if there is better ways to do it. :)